charts/core/src/ChartTooltip/CustomTooltip/CustomTooltip.tsx

```diff
@@ -42,7 +42,7 @@ export function CustomTooltip({
 }: CustomTooltipProps) {
   const { theme } = useDarkMode(darkMode);
 
-  if (seriesData.length === 0 || !seriesData[0].data[0]) {
+  if (seriesData.length === 0 || seriesData[0].data[0] == undefined) {
```

Comment:
Why does negation not work here? (and why `==` instead of `===`?)

Reply:
Got it! Small nit then: I believe we have a `isDefined` util in lib. The distinction between `==` and `===` isn't always immediately obvious, so I've been recommending to check `!isDefined` instead.
Not a blocker though